### PR TITLE
Added phpdoc to pipeline variable

### PR DESCRIPTION
**Pull request by Aliance**

### CODE REVIEW

**Comment by nrk** [comment]
Isn't @var ClientContextInterface enough since it's already referred to the variable $pipeline set below the phpdoc block? Honest question, my phpdoc-fu is admittedly weak :-)

**Comment by Aliance** [comment]
Without this changes PHPStorm highlighted methods, called from `$pipeline` variable.

File: src/Client.php [hunk 1]
```diff
@@ -433,8 +433,8 @@ protected function createPipeline(array $options = null, $callable = null)
             $class = 'Predis\Pipeline\Pipeline';
         }
 
-        /*
-         * @var ClientContextInterface
+        /**
```
Comment (GrahamCampbell) [inline src/Client.php]: This change is important. `/*` is VERY different from `/**`. PHP even tokenizes it differently.

Comment (nrk) [inline src/Client.php]: Yup that was just an oversight on my part, my doubt was about the addition of `$pipeline`.

Comment (GrahamCampbell) [inline src/Client.php]: the `$pipeline` thing is wrong, yeh

Comment (nrk) [inline src/Client.php]: Alright then, fixed without merging this PR (but put a note in the commit message to thank @Aliance for spotting this one).

**Pull request closed without merging**